open-media-match/src/OpenMediaMatch/app.py

```diff
@@ -54,6 +54,10 @@ def index():
             "index.html.j2", production=app.config.get("PRODUCTION")
         )
 
+    @app.route("/hello")
+    def hello_world():
+        return "Hello, world!\n"
+    
```

Comment:
Blocking: Remove this code from the hello world example

Reply:
Done